README.md

```diff
@@ -532,6 +552,37 @@ Breaking changes are changes that could break a client that is relying on the Op
 [See some examples of breaking and non-breaking changes](breaking-changes.md).  
 Note: this is a Beta feature. Please report issues.
 
+### Breaking Changes Checks
+The new way do detect breaking changes in specifications. It works differently form the original "Breaking changes" feature.
+
+The Original breaking changes feature tries to 
+remove non-breaking diffs during diff construction, and reports any other changes as breaking.
+This behaviour is not suitable for regular strict validation for complex APIs.
+
+That's why the breaking changes checks feature is here.
+
+It uses the slice of checks which are used to check for backward compatibility. Each check is like a rule that checks specific change type.
+It operaties on the API level (path+operation).
```

Comment:
I removed the technical implementation details because they are only relevant to developers and the readme is for general users

Reply:
Thanks for docs improvements you've made